.github/commitlint.config.mjs

```diff
@@ -16,7 +16,7 @@ const validateBodyMaxLengthIgnoringDeps = async (parsedCommit) => {
   ]
```

Comment:
What is ES in the context of the commit message? Please make commits actually useful.

Reply:
>ES Modules was introduced in ECMAScript 6 in 2015. It was created to provide a standardized way to modularize JavaScript code. ES Modules is now widely supported by modern web browsers and Node.js.

I thought it would be easy to verify what "ES module" means related to a javascript file change with a quick google search.